.eslintrc

```diff
@@ -49,7 +49,8 @@
             {"before": false, "after": true}
         ],
         "complexity": [
-          2
+          2,
+          {"max": 21}
```

Comment:
Yeah, in one of the reducers. My next big goal is to unwind the ball-of-mud `case` statements into the better structure `redux-actions` makes available.